bears/vcs/git/GitCommitBear.py

```diff
@@ -41,13 +50,37 @@ def get_body_checks_metadata(cls):
             omit={'self', 'body'})
 
     @classmethod
+    def get_issue_checks_metadata(cls):
+        return FunctionMetadata.from_function(
+            cls.check_issue,
+            omit={'self', 'body'})
+
+    @classmethod
     def get_metadata(cls):
         return FunctionMetadata.merge(
             FunctionMetadata.from_function(
                 cls.run,
                 omit={'self', 'dependency_results'}),
             cls.get_shortlog_checks_metadata(),
-            cls.get_body_checks_metadata())
+            cls.get_body_checks_metadata(),
+            cls.get_issue_checks_metadata())
+
+    @staticmethod
+    def get_host_from_remotes():
+        """
+        Retrieve the host from supplied git remotes.
```

Comment:
I think you should point out in the docs that you return the first remote that appears in `git remotes` :)